pkg/flink/internal/store/store_utils.go

```diff
@@ -388,6 +391,25 @@ func startsWithValidSQL(statement string) bool {
 	return config.SQLKeywords.Contains(firstWord)
 }
 
+func isUserSecretKey(key string) bool {
+	parts := strings.Split(key, ".")
+	count := len(parts)
+
+	if count < 1 {
+		return false
+	}
+
+	last := strings.ToLower(parts[count-1])
+
+	// add *ecrets, s*crets and so on. Maybe not * but only two letter that are wrong after the s? hmm
+	matches, err := regexp.MatchString(".*?ecret|s.{0,2}?cret|se.{0,2}?ret|sec.{0,2}?et|secr.{0,2}?t|secre.{0,2}?", last)
```

Comment:
can you see if there's a levenshtein distance algo somewhere for go (or just build it yourself as a util). Then you'll also cover mishits on keyboard. Let's say score of 2 is what you want

Reply:
lemme look into that

Reply:
Neat pointer. Thanks!